src/components/Map/index.tsx

```diff
@@ -44,6 +50,7 @@ export const Map: FC<ViewportType> = ({
         setViewport(nextViewport)
       }
     >
+      <MapControls isMobile={isMobile} />
```

Comment:
I would argue that we should not add the controls into the map by default. Perhaps in the future we want a map without controls. Instead I would suggest to add the controls in the `RefreshmentMap`, ideally utilizing the `FullScreenMapWrapper` component and its corner slots.